third_party/move/move-core/types/src/effects.rs

```diff
@@ -188,16 +198,16 @@ impl AccountChangeSet {
     }
 }
 
-// TODO: ChangeSet does not have a canonical representation so the derived Ord is not sound.
+// TODO: Changes does not have a canonical representation so the derived Ord is not sound.
 
 /// A collection of changes to a Move state. Each AccountChangeSet in the domain of `accounts`
 /// is guaranteed to be nonempty
 #[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd)]
-pub struct ChangeSet {
-    accounts: BTreeMap<AccountAddress, AccountChangeSet>,
+pub struct Changes<M, R> {
```

Comment:
these single letter is not very easy to understand, I'd prefer just using the full word Module/Resource as generics, wdyt